dashboard/config/locales/slides.en.yml

```diff
@@ -2451,3 +2451,76 @@ en:
       9:
         image: 'notes/mc_2018_closing_9.png'
         text: "You've certainly proven yourself an intrepid coder by now, and I can’t wait to see what you build. And hopefully your adventures with code don't end here. Bon Voyage!"
+    Teaching CS Fundamentals: How the Course Works:
```

Comment:
The other entries at this indentation-level `look_like_this`.. why is this one different?

Reply:
Another good question.  The content editor accidentally made the title the key.  I have been investigating ways to change the key to snake case.